addon/package.json

```diff
@@ -39,6 +39,13 @@
   },
   "preferences": [
     {
+      "name": "NUM_STARS",
```

Comment:
Is this a thing users should be able to change? If not, it shouldn't appear in preferences. Seems like maybe it should just be a constant in the addon code